codes/c/chapter_hashing/hash_map_open_addressing.c

```diff
@@ -0,0 +1,214 @@
+/**
+ * File: hash_map_open_addressing.c
+ * Created Time: 2023-10-6
+ * Author: lclc6 (w1929522410@163.com)
+ */
+
+#include "../utils/common.h"
+
+/* 开放寻址哈希表 */
+struct Pair {
+    int key;
+    char *val;
+};
+
+typedef struct Pair Pair;
+
+struct HashMapOpenAddressing {
```

Comment:
I noticed that some structs are named in camelCase, while others are PascalCase. 

Which one is more popular? @Gonglja